debian/changelog

```diff
@@ -0,0 +1,5 @@
+python-ev3dev (0.2.2.post1-1) unstable; urgency=low
+
+  * source package automatically created by stdeb 0.6.0+git
+
+ -- Denis Demidov <dennis.demidov@gmail.com>  Mon, 28 Sep 2015 11:15:43 +0200
```

Comment:
The name and email here is whomever did the packaging, not necessarily the author of the software.